exporter/exporterhelper/internal/bounded_memory_queue.go

```diff
@@ -65,11 +44,19 @@ func (q *boundedMemoryQueue) Offer(ctx context.Context, req any) error {
 	}
 }
 
+// Poll returns a request from the queue once it's available. It returns false if the queue is stopped.
+func (q *boundedMemoryQueue) Poll() (QueueRequest, bool) {
+	item, ok := <-q.items
+	if ok {
+		return item, true
+	}
+	return QueueRequest{}, false
```

Comment:
Why checking for ok, instead of returning `item, ok`? Or directly `<-q.items`?

Reply:
good point 👍